packages/react-components/src/templates/TeamProfileAbout.tsx

```diff
@@ -49,36 +47,13 @@ const TeamProfileAbout: React.FC<TeamProfileAboutProps> = ({
         expertiseAndResourceTags={expertiseAndResourceTags}
       />
     ) : null}
-    {inactiveSince ? (
-      <TeamMembersTabbedCard title="Team Members" members={members} />
-    ) : members.length ? (
-      <section id={teamListElementId}>
-        <TeamMembersSection
-          members={members.map(
-            ({
-              displayName,
-              role,
-              firstName,
-              lastName,
-              avatarUrl,
-              id,
-              labs = [],
-            }) => ({
-              firstLine: displayName,
-              secondLine: role,
-              thirdLine: getUniqueCommaStringWithSuffix(
-                labs.map((lab) => lab.name),
-                'Lab',
-              ),
-              avatarUrl,
-              firstName,
-              lastName,
-              id,
-            }),
-          )}
-        />
-      </section>
-    ) : null}
+    <section id={teamListElementId}>
+      <TeamMembersTabbedCard
```

Comment:
I noticed that this problem where the tabbed card occupies more space than it should it's happening again. I think it's maybe due to this  `section` that is wrapping `TeamMembersTabbedCard`. Is it needed? :)
![2239 hub asap science_network_teams_8c88a42b-9fc7-4ec1-8606-204be75d252c_about(iPhone SE)](https://user-images.githubusercontent.com/16595804/199260926-60d380bb-5978-4c57-9d97-3abd45434a02.png)

Reply:
Kept the section but passed the overflow style, it needed the id because there is a functionality which moves the screen to the members section upon clicking the +X members in the header.

Reply:
nice! It makes sense :)